core/src/main/resources/reference.conf

```diff
@@ -558,10 +558,8 @@ services {
       # performance but will both lead to a higher memory usage as well as increase the risk that metadata events
       # might not have been persisted in the event of a Cromwell crash.
       #
-      # For normal usage the default value of 1 (effectively no batching) should be fine but for larger/production
-      # environments we recommend a value of at least 500. There'll be no one size fits all number here so we recommend
-      # benchmarking performance and tuning the value to match your environment
-      # db-batch-size = 1
+      # This value may require benchmarking performance and tuning the value to match your environment.
```

Comment:
Just wear gloves to be on the safe side.

Yeah, I have no idea what significance that number held but I ran too many tests for that to have been completely made up.